docs/minio-job.md

```diff
@@ -0,0 +1,209 @@
+# MinioJob is a Kubernetes Job that runs mc commands
+here is an example of a MinioJob:
+```yaml
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: mc-job-sa
+---
+apiVersion: sts.min.io/v1alpha1
+kind: PolicyBinding
+metadata:
+  name: mc-job-binding
+spec:
+  application:
+    serviceaccount: mc-job-sa
+  policies:
+    - consoleAdmin
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: mytestsecret
+data:
+  PASSWORD: cGVkcm8xMjM=
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: mytestsecretenvs
+data:
+  USER: ZGFuaWVs
+  PASSWORD: ZGFuaWVsMTIz
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: mytestconfig
+data:
+  policy.json:  |
+    {
+        "Version": "2012-10-17",
+        "Statement": [
+            {
+                "Effect": "Allow",
+                "Action": [
+                    "s3:*"
+                ],
+                "Resource": [
+                    "arn:aws:s3:::memes",
+                    "arn:aws:s3:::memes/*"
+                ]
+            }
+        ]
+    }
+---
+apiVersion: job.min.io/v1alpha1
+kind: MinIOJob
+metadata:
+  name: minio-test-job
+spec:
+  mcImage: quay.io/minio/mc:latest # optional, defaults to minio/mc:latest
```

Comment:
The default is now `quay.io/minio/mc:latest` (or even a fixed version that would be my preference). I would suggest to include this line as a comment, because we don't want users to change this (only in very specific scenarios).
```suggestion
  #mcImage: quay.io/minio/mc:latest
```